ROOTFS/etc/waggle/sanity/warning/check_hanwha_camera.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python3
+
+import sys
+import json
+import urllib.error, urllib.request
+import time
+
+def read_camera_manifest(path='/etc/waggle/node_manifest.json'):
+    with open(path, 'r') as file:
+        manifest = json.loads(file.read())
+        if 'cameras' not in manifest:
+            raise Exception(f'No cameras in the manifest: {path}')
+        else:
+            for camera in manifest['cameras']:
+                if camera['present'] == True:
+                    yield camera
+
+
+def http_request(url, user='waggle', pw='Waggle1@34', timeout=2):
+    passman = urllib.request.HTTPPasswordMgrWithDefaultRealm()
+    passman.add_password(None, url, user, pw)
+    authhandler = urllib.request.HTTPDigestAuthHandler(passman)
+    opener = urllib.request.build_opener(authhandler)
+    urllib.request.install_opener(opener)
+    headers = {'Accept': 'application/json'}
+    req = urllib.request.Request(url, headers=headers)
+    try:
+        with urllib.request.urlopen(req, timeout=timeout) as response:
+            return response.code, response.read()
+    except urllib.error.URLError:
+        return 404, b''
+
+
+def check():
+    code_when_exit = 0
+    cameras = list(read_camera_manifest())
+    # if no camera listed, we do not check
+    if len(cameras) == 0:
+        print('No cameras listed in the manifest. Exiting...')
+        exit(code_when_exit)
+
+    for n in range(10, 20, 1):
+        # get device info
+        return_code, body_json = http_request(f'http://10.31.81.{n}/stw-cgi/system.cgi?msubmenu=deviceinfo&action=view')
```

Comment:
Instead of hard-coding the '10.31.81.' everywhere please pull the value from the /etc/waggle/config.ini file

```
[network]
static-ip-nx = 10.31.81.1
```

```
def read_config_section_dict(filename, section):
    config = configparser.ConfigParser()

    if not config.read(filename):
        logging.warning(f"could not read config file {filename}")
        return {}

    try:
        return dict(config[section])
    except Exception:
        logging.warning("could not read config section [%s]", section)

    return {}

ipaddr = read_config_section_dict("/etc/waggle/config.ini", "network")[static-ip-nx, "10.31.81.1"]
# get "10.31.81" part
ipaddr = ipaddr.rsplit(".", 1)[0]

```

see: https://github.com/waggle-sensor/waggle-network-watchdog/blob/main/ROOTFS/usr/bin/waggle_network_watchdog.py#L85

Reply:
Sure. I will make it a variable within the file so if we need to change it, it would be easier to do so.